internal/services/gateway/action/project.go

```diff
@@ -200,20 +203,28 @@ func (h *ActionHandler) UpdateProject(ctx context.Context, projectRef string, re
 	if req.PassVarsToForkedPR != nil {
 		p.PassVarsToForkedPR = *req.PassVarsToForkedPR
 	}
+	if req.MembersCanPerformRunActions != nil {
+		p.MembersCanPerformRunActions = *req.MembersCanPerformRunActions
+	}
+
+	if p.OwnerType == cstypes.ObjectKindUser && p.MembersCanPerformRunActions {
+		return nil, util.NewAPIError(util.ErrBadRequest, errors.Errorf("can't set MembersCanPerformRunActions true for user projects"))
```

Comment:
This check should be done in the configstore. (Like already done above in this patch, also if in the wrong place).